code/modules/clothing/masks/transmog.dm

```diff
@@ -17,7 +17,8 @@
 		/obj/item/stack/sheet/animalhide/monkey	=	/obj/item/clothing/mask/morphing/monkey,
 		/obj/item/stack/sheet/animalhide/lizard	=	/obj/item/clothing/mask/morphing/lizard,
 		/obj/item/stack/sheet/animalhide/xeno	=	/obj/item/clothing/mask/morphing/xeno,
-		/obj/item/stack/sheet/animalhide/human	=	/obj/item/clothing/mask/morphing/human)
+		/obj/item/stack/sheet/animalhide/human	=	/obj/item/clothing/mask/morphing/human,
+		/obj/item/weapon/ectoplasm				=	/obj/item/clothing/mask/morphing/ghost)
```

Comment:
Nah, it's because you used tabs instead of spaces for alignment.

The rule of thumb is as follows:
You indent things to a certain indentation level with tabs.

You align things on the same indentation level with spaces.